lib/models/cac_location.rb

```diff
@@ -53,8 +53,11 @@ class CACLocation
   #
   # @return [Array] all CACLocations from the API
   def self.all_from_api
+    arcgis_locations = ArcGISClient.all_features
     TestSites::CAC.cac_data.map do |hash|
-      CACLocation.new(hash)
+      CACLocation.new(hash).tap do |location|
+        location.arcgis_location = arcgis_locations[location.esri_global_id]
```

Comment:
If `arcgis` and `esri` are equivalent, can we just use one or the other term and not both? Esri is simpler and easier to say so I'd favor that, but then we'd need a global rename on filenames too.

Reply:
ARCGiS is the product, ESRI is the company that makes it. I think it's better to use ARCGiS.

Reply:
I know that, you know that, but will new team members looking at this code for the first time? My goal here is readability and reducing cognitive load. Choosing names are an easy win for both.

Reply:
I had already made the change. See https://github.com/Spark-Bio/covid-storepoint/pull/52/commits/34b8da7f8c3b639deecbcbf1a4f647bd56041575

Reply:
Yeah, didn’t catch it till after, my bad!